Add note for end of feature development (#962)
pimcore/admin-ui-classic-bundle: README.md

# Pimcore Admin Classic Bundle

> [!IMPORTANT]

> > This bundle as reached **end of feature development.** We will no longer

Reviewer: ```suggestion
> This bundle has reached **end of feature development.** We will no longer

```

# Pimcore Admin Classic Bundle

> [!IMPORTANT]
> This bundle has reached **end of feature development.** We will no longer

> accept or review any pull requests related to new features or improvements for this bundle.
>

> > However, bug fixes will continue to be accepted, reviewed, and merged at least until the end of 2025.

Reviewer: ```suggestion
> However, we will continue to accept, review and merge bug fixes at least until the end of 2025.

```
A bit petty, but it might be better to restructure this sentence a bit.

# Pimcore Admin Classic Bundle

> [!IMPORTANT]
> This bundle has reached **end of feature development.** We will no longer

> 

Reviewer 2: ```suggestion
```

> accept or review any pull requests related to new features or improvements for this bundle.
>
> However, we will continue to accept, review and merge bug fixes at least until the end of 2025.

> 

Reviewer 2: ```suggestion
```
